media/js/mozorg/home/home.js

```diff
@@ -66,4 +70,66 @@
 
     // Video card interactions.
     initVideoEvents();
-})();
+
+    /*
+     * Sticky CTA
+     */
+
+
+    var stickyCTA = document.getElementById('download-firefox-sticky-cta');
+    $(stickyCTA).attr('aria-hidden', 'true');
```

Comment:
Could just do the jQuery-fication once to avoid multiple `$` wrappings/DOM lookups:

`var $stickyCTA = $('#download-firefox-sticky-cta');`